docs/core/testing/unit-testing-published-output.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Test Published Output with dotnet vstest
+description: Learn how to run tests on published output with the dotnet vstest command.
+keywords: MSTest, .NET, .NET Core, xUnit, test, published, output, test after publishing, run tests on dll
+author: kehavens
+ms.author: kehavens
+ms.date: 10/18/2017
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+ms.assetid: 3965e4ca-75b8-4969-b3af-ca993c397a15
+---
+
+# Test Published Output with dotnet vstest
+
+You can run tests on already published output by using the `dotnet vstest` command. This will work on both xUnit and MSTest tests. Simply locate the DLL file that was part of your published output and run:
+```
+dotnet vstest <MyPublishedTests>.dll
```

Comment:
This will work if test project is targeting framework `netcoreapp`. If project will target framework `net4*` then user has to explicitly pass `--Framework`
Example:
```
dotnet vstest <MyPublishedTests>.dll  --Framework:".NETFramework,Version=v4.6"
```

for more info you can refer `dotnet vstest -?`

Reply:
The dependency of passing  `--Framework` has been resolved it Visual Studio 2017 update5. Now vstest will automatically figure out framework. It this docs is for Update5 then you can ignore my last comment.

Reply:
Great additional! I'll add a note or section on this.